mne/commands/tests/test_commands.py

```diff
@@ -202,4 +203,37 @@ def test_watershed_bem():
         mne_watershed_bem.run()
 
 
+@slow_test
+@requires_mne
+@requires_freesurfer
+@sample.requires_sample_data
+def test_flash_bem():
```

Comment:
Yeah. I can run the `mri_ms_fitparms` command. Looks like I'm on `freesurfer-Linux-centos6_x86_64-stable-pub-v5.3.0` if it matters.